src/Microsoft.AspNetCore.Mvc.Analyzers.Experimental/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers
+{
+    public static class DiagnosticDescriptors
+    {
+        public static readonly DiagnosticDescriptor MVC7000_ApiActionsMustBeAttributeRouted =
```

Comment:
This is primarily a rename. Also changing the diagnostic Ids to 7000 so that we have enough space for regular analyzers.